content/en/docs/Usage/backup.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Automated Backup
+linkTitle: Automated Backup
+date: 2024-12-21
+description: >
+  Information on the in-built backup system
+---
+
+Navidrome version 0.54.x introduces a backup feature that allows you to safeguard your music server's data. This guide will walk you through configuring backups using both the config.toml file and environment variables, as well as where to locate the backups.
+
+## Configuring Backup with config.toml
+
+To configure backups using the navidrome.toml file, insert the following lines to set up backups:
+
+~~~conf
+BackupFolder = "/path/to/backup/folder"
+BackupSchedule = "0 0 * * *"
```

Comment:
Please use the proper values from https://www.navidrome.org/docs/usage/configuration-options/#:~:text=album%2Fartist.*%2C%20external-,backup.path,-ND_BACKUP_PATH

Note that count is required (all three properties are required)

As a nested toml example
```toml
[Backup]
Path = "/path/to/backup/folder"
Count = 5
Schedule =  "0 0 * * *"
```

Or `Backup.Path`. Also please not it in docker